validator/valnode/redis/consumer.go

```diff
@@ -42,12 +45,56 @@ func NewValidationServer(cfg *ValidationServerConfig, spawner validator.Validati
 		}
 		consumers[mr] = c
 	}
+	var (
```

Comment:
I don't like this happening in New.
1) you don't have a context here
2) if starting a full nitro node that also has this enabled for some reason - the node will appear stuck and do nothing for some time.
One way would be to create an initialize function which has a context.. but I actually think best solution would be in the per-wasm-module-root thread in Start.
It would make exiting after timeout passed harder.. I'm fine with every thread just doing a loop waiting for a redis queue until there is one, and the first thread printing an error every timeout if no thread connected

Reply:
Done.